pulp_container/app/models.py

```diff
@@ -320,26 +320,28 @@ class ContainerRemote(Remote, AutoAddObjPermsMixin, AutoDeleteObjPermsMixin):
         upstream_name (models.CharField): The name of the image at the remote.
         include_foreign_layers (models.BooleanField): Foreign layers in the remote
             are included. They are not included by default.
+        include_tags (fields.ArrayField): List of tags to include during sync.
+        exclude_tags (fields.ArrayField): List of tags to exclude during sync.
+        sigstore (models.TextField): The URL to a SigStore where signatures of container images
+            should be synced from.
     """
 
     upstream_name = models.CharField(max_length=255, db_index=True)
     include_foreign_layers = models.BooleanField(default=False)
     include_tags = fields.ArrayField(models.CharField(max_length=255, null=True), null=True)
     exclude_tags = fields.ArrayField(models.CharField(max_length=255, null=True), null=True)
+    sigstore = models.TextField(null=True)
```

Comment:
I actually wanted to make it the same as in pulpcore, TextFiled. But you bring a good point, I added the question to the open floor agenda.

Reply:
Info from @fao89: we had an OpenAPI issue: https://github.com/pulp/pulpcore/pull/919
but we killed the OpenAPI schema view, so we can use it again.

It was discussed at the open floor today that we do not want to change it in pulpcore.
I would keep TextField since I'm not sure which limit to set for the length, and pulpcore has unlim.

If you feel strongly about this, feel free to let me know before we merge the `signing` branch into the main (probably not any time soon :))

Reply:
We can go with `TextField` for now. Thank you for adding the validation.